app/services/anno_auth_validator.rb

```diff
@@ -0,0 +1,46 @@
+require 'json'
+require 'net/http'
+require 'jwt'
+
+class AnnoAuthValidator
+
+    def self.authorize(*params)
```

Comment:
Two space indentation please, per https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md

Reply:
Updated. For some reason for a new Ruby file, VSCode set the tabbing wrong.